pages/models/generations/SM1/index.md

```diff
@@ -0,0 +1,293 @@
+
+The stator-rotor inductance also depends on the electrical angle $$\theta_{me}$$ between the rotor winding axis and the stator winding axis. The value of this inductance for phase $$a$$ can be calculated with the expression $$\mathcal{L}_{af} = \mathcal{L}_{fa} = L_{af}\cos(\theta_{me})$$, while the same expression can be applied for phase $$b$$ and $$c$$ but replacing $$\theta_{me}$$ with $$\theta_{me} - 120º$$ and $$\theta_{me} + 120º$$ respectively.
+
+### Operational limits
+
+Depending on the difference in phase between the rotor and grid rotation, the power transferred will be composed of active ($$P$$) and reactive ($$Q$$) power.
+
+These generated powers are limited by the heat limits of the components of the generator. The limits considered are the Armature and field current limit, due to Joule effect heating of both windings, and end region heating limit, which occurs due to currents in the structure of the stator when the field is underexcited.
+
+The following charts show the capability and compound curves for different power factors[[1]](#1).
+
+<p float="center">
+<img src="{{'/pages/models/generations/SM1/SM_CapCurve.svg' | relative_url }}"
+     alt="Capability curve of a Synchronous Generator"
+     width="350px"/>
+<img src="{{'/pages/models/generations/SM1/SM_CompCurves.svg' | relative_url }}"
+     alt="Compound curves of a Synchronous Generator"
+     width="400px"/>
+</p>
+<div style="text-align:center">
+Figures 5 and 6: Capability and compound curves of a synchronous machine
+</div>
+<br>
+
+The left chart shows the $$P$$ and $$Q$$ maximum values for different limit curves, which depend on the refrigerator used, while the right chart shows the dependency on exciter current and apparent power for different values of the power factor.
+
+## Derived models
+
+The model can be solved by performing some additional assumptions and transformations. Here some submodels that have arised from different works are listed, with its particularities:
+
+- **GENSAL/GENROU without saturation**: This model is described in Sauer and Pai formulation [[6]](#6), and is valid for both round-rotor and salient-poles synchronous machine.
+
+- **GENROU with saturation**: This model is an extension of the previous one, considering the saturation of the magnetic circuit.
+
+- **GENSAL with saturation**: Same as the previous model, but in this case it is treated slightly different due to the saturation occuring only in the salient-poles axis only, while the previous model considers saturation along both *d* and *q* axis.
+
+- **GENTPF/GENTPJ**: It models the saturation in a different way, including the saturation in the network interface equations, something that the GENSAL/GENROU models do not consider, and it reflects better the saturation effects in the whole machine.
+
+Details can be found here:
+
+{% include subfolderNav.html %}
+
+## Open source implementations
```

Comment:
Can you check this?
![image](https://github.com/CRESYM/colib0.github.io/assets/157120809/a5ec370c-a590-4c03-89b7-ad34b737d477)

Reply:
Added a missing line break